tests/web/rpc/v1/test_storage_routing.py

```diff
@@ -170,10 +184,16 @@ def _output_metrics(self, routing_context: RoutingContext) -> None:
     timer=Timer("stuff"),
     query_result=MagicMock(spec=QueryResult),
     extra_info={},
+    query_id=uuid.uuid4().hex,
+    tenant_ids={
+        "organization_id": 1,
+        "referrer": "something",
+    },
 )
 
 
 @pytest.mark.redis_db
+@pytest.mark.clickhouse_db
```

Comment:
```suggestion
@pytest.mark.eap
```

is better to use nowadays, much faster